File: HelloJava/src/day210210/MyDelete.java
package day210210;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;
import java.sql.Statement;

public class MyDelete {
	public static void main(String[] args) throws ClassNotFoundException, SQLException {
		Connection con;
		String url = "jdbc:oracle:thin:@localhost:1521:xe";
		Class.forName("oracle.jdbc.driver.OracleDriver");

		con = DriverManager.getConnection(url, "python", "python");

		String sql = "DELETE FROM SAMPLE WHERE COL01 = '1'";
		Statement stmt = con.createStatement();

		int result = stmt.executeUpdate(sql);
		if (result > 0) {
			System.out.println("성공");
		} else {
			System.out.println("실패");
		}
		stmt.close();
		con.close();
	}
}

File: HelloPython/d210210/mydelete.py
import cx_Oracle

conn = cx_Oracle.connect('python/python@localhost:1521/xe')

cur = conn.cursor()

sql = "delete from sample where col01 = :col01"
V_col01 = '95'
params = (V_col01,)
cur.execute(sql, params)
print(cur.rowcount)

conn.commit()

cur.close()
conn.close()
